src/app/Dashboard/AddCard.tsx

```diff
@@ -194,7 +196,7 @@ export const AddCard: React.FC<AddCardProps> = (_) => {
                   </Select>
                   <Text>
                     {selection
-                      ? getConfigByTitle(selection).descriptionFull
+                      ? t(getConfigByTitle(selection, t).descriptionFull as string)
```

Comment:
Any issue that we need to type assertion here?

Reply:
Ahh maybe we should check if description is string => its a key and pass to `t`. Or else just display it.